ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -760,6 +763,24 @@ open class DrawerController: UIViewController {
         originalContentOffsetY = contentScrollView.contentOffset.y
     }
 
+    /// To disimiss presentingViewController
+    /// - Parameters:
+    ///   - animated: dimiss should happen with animation or not
+    ///   - isResizing: flag indicating wheter dismiss is being called due to user resizing the drawer, default is false
+    /// - Returns: whether drawer got dismissed or not
+    private func dismissPresentingViewController(animated: Bool, isResizing: Bool = false) -> Bool {
```

Comment:
@discardableResult in order to avoid the "_ =" in places you don't use the result?

Reply:
Added.